changelog/fix-increase-docker-php-memory-limit

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: dev
+
+Increased the PHP memory limit for the Docker container
```

Comment:
I think that you should be able to add a changelog file with a comment, rather than an entry. This is not a part of the plugin. Selecting `patch` and then `fix` would allow you to continue without an entry.

Reply:
Ok, I wasn't sure if we should mention it as a dev entry in the changelog, but I've changed it.